reference/method-patterns.md

```diff
@@ -29,19 +29,27 @@ The return type of a method is _not_ represented in method patterns. Methods in
 
 ### Examples
 
-So if you wanted to match invocations of `String.toString(int beginIndex)` then this expression would match only that method: `java.lang.String toString(int)`. If you wanted to match the two-argument form of the same
-
-| Method Pattern | Matches |
-| :--- | :--- |
-| `java.lang.String substring(int)` | Exactly the [single argument overload](https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/lang/String.html#substring%28int%29) of `String.substring()` |
-| `java.lang.String substring(int, int)` | Exactly the [two argument overload](https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/lang/String.html#substring%28int,int%29) of `String.substring()` |
-| `java.lang.String substring(..)` | Any overload of `String.substring()` |
-| `java.lang.String *(int)` | Any method on `String` that accepts a single argument of type `int` |
-| `com.yourorg.Foo bar(int, String, ..)` | Any method on `Foo` named `bar` accepting an `int`, a `String`, and zero or more other arguments of any type |
-| `com.yourorg.Foo <constructor>(*, *, *)` | Constructors of `Foo` accepting exactly three arguments of any type. |
-| `*..String *(..)` | Any method accepting any arguments on classes named "String" in any package |
-| `*..* *(..)` | Any method accepting any arguments on any class |
-| `org.example..* *(..)` | Any method on any class in the "org.example" package, or any sub-package of "org.example" |
+So if you wanted to match invocations of `String.substring(int beginIndex)` then this expression would match only that method: `java.lang.String substring(int)`.
+If you wanted to match the two-argument form of the same, you can use the method pattern `java.lang.String substring(int, int)`.
```

Comment:
There seems to have been a half-finished sentence there; Wrapped it up as best as I could, and swapped out `toString` for `substring`, as `toString` does not take any `int` arguments.

My suggestion though would be to just drop these two sentences and instead introduce the table, if we even do that at all.
```suggestion
```

Reply:
Making the table makes sense to me.

Reply:
Oh the table was already there, I merely expanded it, which kind of butchered the diff.

I'll remove the introduction sentences, which were half finished to begin with, and duplicate the table. :)